US-CBP/cbp-theme
# Interactivity page

#### What's this PR do?

1. Implemented content for interactivity. 
2. Changed padding for wrapper. (swapped padding from being called in `ds-common-styles.scss` to `layout-development.scss`)
3. Removed unused sub-headers for the sidenav. 
4. Added image component with gatsby-image. 
5. Slight modification to text margins.

#### Where should the reviewer start?
`interactivity.js` for the new content page
`ds-ux-guidelines/src/ds-components/image.js` for new image component
`ds-ux-guidelines/src/ds-common-styles/ds-common-styles.scss` for text spacing changes

#### How should this be manually tested?
Manually navigate to the Interactivity section to make sure content is displayed properly. Check other foundation sections to see if layouts are displayed properly.

#### Any background context you want to provide?
I am currently in the process of modifying the tabs into its own component, as well as modifying all current image to use the image component.

#### What are the relevant tickets?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13155212/84076475-cc441a00-a9a3-11ea-9b7c-78ab4c6e2eca.png)

#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
